docs/content/en/docs/references/cli/_index.md

```diff
@@ -72,7 +72,8 @@ Flags:
       --insecure-registry strings    Target registries for built images which are not secure
   -n, --namespace string             Run deployments in the specified namespace
       --no-prune                     Skip removing images and containers built by Skaffold
-  -o, --output *flags.TemplateFlag   Used in conjuction with --quiet flag. Format output with go-template. For full struct documentation, see https://godoc.org/github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd#BuildOutput (default {{.}})
+  -o, --output *flags.TemplateFlag   Used in conjuction with --quiet flag. Format output with go-template. For full struct documentation, see https://godoc.org/github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd#BuildOutput (default {{ range .Builds}}{{if not .}}{{else}}{{.Target.ImageName}} -> {{if .Error}}Error: {{.Error}}{{else}}{{.Result.Tag}}{{end}}{{end}}
```

Comment:
This struct no longer exists 
https://godoc.org/github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/cmd#BuildOutput
it should point to the newly added result struct.